projects/1.18/assets/terraincognita/terraincognita/lang/zh_cn.json

```
@@ -23,40 +26,35 @@
   "block.terraincognita.potted_globeflower": "金凤花盆栽",
   "block.terraincognita.marigold": "万寿菊",
   "block.terraincognita.potted_marigold": "万寿菊盆栽",
-  "block.terraincognita.blue_lupine": "蓝色羽扇豆",
-  "block.terraincognita.potted_blue_lupine": "蓝色羽扇豆盆栽",
-  "block.terraincognita.gladiolus": "唐菖蒲",
-  "block.terraincognita.potted_gladiolus": "唐菖蒲盆栽",
   "block.terraincognita.snapdragon": "金鱼草",
   "block.terraincognita.potted_snapdragon": "金鱼草盆栽",
-  "block.terraincognita.geranium": "老鹳草",
-  "block.terraincognita.potted_geranium": "老鹳草盆栽",
   "block.terraincognita.edelweiss": "雪绒花",
   "block.terraincognita.potted_edelweiss": "雪绒花盆栽",
   "block.terraincognita.alpine_pink": "高山石竹",
   "block.terraincognita.potted_alpine_pink": "高山石竹盆栽",
-  "block.terraincognita.saxifrage": "虎耳草",
-  "block.terraincognita.potted_saxifrage": "虎耳草盆栽",
+  "block.terraincognita.yellow_saxifrage": "黄色虎耳草",
+  "block.terraincognita.potted_yellow_saxifrage": "盆栽黄色虎耳草",
+  "block.terraincognita.magenta_saxifrage": "品红色虎耳草",
+  "block.terraincognita.potted_magenta_saxifrage": "品红色虎耳草盆栽",
   "block.terraincognita.gentian": "龙胆花",
   "block.terraincognita.potted_gentian": "龙胆花盆栽",
+  "block.terraincognita.aster": "紫菀",
+  "block.terraincognita.potted_aster": "紫菀盆栽",
   "block.terraincognita.blue_iris": "蓝色鸢尾花",
   "block.terraincognita.potted_blue_iris": "蓝色鸢尾花盆栽",
   "block.terraincognita.purple_iris": "紫色鸢尾花",
   "block.terraincognita.potted_purple_iris": "紫色鸢尾花盆栽",
   "block.terraincognita.black_iris": "黑色鸢尾花",
   "block.terraincognita.potted_black_iris": "黑色鸢尾花盆栽",
-  "block.terraincognita.dwarf_fireweed": "柳兰花",
-  "block.terraincognita.potted_dwarf_fireweed": "柳兰花盆栽",
-  "block.terraincognita.arctic_poppy": "冰岛虞美人",
-  "block.terraincognita.potted_arctic_poppy": "冰岛虞美人盆栽",
+  "block.terraincognita.heather": "石楠花",
+  "block.terraincognita.potted_heather": "石楠花盆栽",
```

```suggestion
  "block.terraincognita.heather": "帚石南",
  "block.terraincognita.potted_heather": "帚石南盆栽",
```
http://www.iplant.cn/info/Calluna%20vulgaris